src/parser/inc/Type.hpp

```diff
@@ -19,6 +19,8 @@ class Type {
     std::string name;
     // If !isPrimitive then NONE
     PRIMITIVE primType;
+
+    int offset();
```

Comment:
Huh?

Reply:
Provides an function to output the number of bits to store a single element of this type.